content/en/events/guides/migrating_to_new_events_features.md

```diff
@@ -20,21 +22,25 @@ This guide describes how to migrate your organization’s event monitors and das
 
 ## Migration process
 
-1. **Request early access.** Events Explorer is in private beta. [Contact support][3] to request access for your organization. The support team sends you a notification when Events Explorer is enabled for your organization. You can access the new Explorer and other features through the Events menu in the main navigation, alongside the existing Stream.
+### Migrate data configured externally (Terraform, API)
+If you manage your Datadog configuration with external, API-based scripts or tools such as Terraform, update your dashboard and monitor definitions in your configuration file. Otherwise the old query syntax is applied again at the next deployment and overwrites updated queries. Review the syntax changes described below and use the [migration script][3] to see what converted queries look like. [Contact Support][4] for assistance at this stage.
 
-   {{< img src="events/guides/navigation.png" alt="Events navigation" style="width:100%;" >}}
+### Datadog migrates data stored in Datadog servers
+Datadog is responsible for migrating data stored in its servers. You do not need to take any action for configuration stored directly in Datadog. Datadog automatically migrates Event data contained in monitors and dashboard widgets, including the event stream, timeline, and overlays.
```

Comment:
to precise:

1) legacy event monitors will be duplicated:
- new monitors based on the new query will be created and automatically enabled.
- existing monitors will be muted. A "(legacy)" prefix will be added to distinguish them.
You can filter the 2 monitor types using the "Event Migration" facet in the monitors list: https://a.cl.ly/rRuNAgm1

SLO and downtimes links will also be updated to point to the new monitors.

2) dashboard widgets using event queries will directly updated.
No new dashboard will be created